pyproject.toml

```
@@ -16,7 +16,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
-qibolab = { git = "https://github.com/qiboteam/qibolab.git", branch = "main" }
+qibolab = { git = "https://github.com/qiboteam/qibolab.git", branch = "dummy2-now-with-couplers!" }
 qibo = "^0.1.9"
```

```suggestion
qibo = "^0.2.0"
```
This should fix tests @Jacfomg

---

Nevermind :(